.github/ISSUE_TEMPLATE/06_feature_request.yml

```diff
@@ -1,7 +1,7 @@
 name: 💡 Feature  Request
 description: Suggest an idea for this project.
-title: "[Feature Request]: "
 labels: ["Feature Request"]
```

Comment:
Was this supposed to be the other way around? (title intact, label removed)

Reply:
I really dislike the title prefixes. But yeah we should probably remove the label. I'm not sure how much we want to have redundant labeling in the transition period.